python/analytics/input.py

```diff
@@ -18,6 +18,8 @@
 import sys
 
 from datetime import datetime
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "lib"))
```

Comment:
This is great, this matches what app developers will need to do which makes for more accurate examples